api_tests/nodes/views/test_node_links_detail.py

```diff
@@ -125,7 +125,7 @@ def test_cannot_delete_if_registration(self):
             pointer_id,
         )
         res = self.app.delete(url, auth=self.user.auth, expect_errors=True)
-        assert_equal(res.status_code, 403)
+        assert_equal(res.status_code, 404)
```

Comment:
There should be an equivalent view. /v2/registrations/`reg_id`/node_links/`link_id`/ is a valid endpoint, but DELETEing should give a 405. We may have a bug where the link is giving you /v2/nodes/ instead of /v2/registrations/ though.